components/developer/gdb/Makefile

```diff
@@ -38,9 +38,9 @@ COMPONENT_BUGDB=	utility/gdb
 TPNO=			17258
 CONFIGURE_DEFAULT_DIRS=no
 
-include ../../make-rules/prep.mk
-include ../../make-rules/configure.mk
-include ../../make-rules/ips.mk
+include ../../../make-rules/prep.mk
```

Comment:
You could use $(WS_TOP) or $(WS_MAKE_RULES). Otherwise, looks good. Please, merge commits.